src/app/room/[slug]/layout.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+import React from "react";
+import Navbar from "../../../ui/navbar";
+import Profile from "../../../ui/Profile";
+import { useSnapshot } from "valtio";
+import { state } from "~/app/state";
+
+export default function playerLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const snap = useSnapshot(state);
+  return (
+    <section className="h-screen px-10 py-14 flex flex-col">
+      <Navbar>
```

Comment:
Why the hell do we need the navbar component??? Pls. remove the navbar component and put the logic into this file!!! Or you have a good reason to do it.